### ISSUE TITLE
TOKENINFO_URL in docker container has excess characters

### ISSUE THREAD

**Issue by GJL** [zalando-stups/taupage#324]
The TOKENINFO_URL env variable is not set correctly in the docker container if taupage is configured with `local_planb_tokeninfo: true`

Example:
```
# run from within the docker container

I have no name!@ip-172-xx-xxx-xxx:/data$ env | grep TOKENINFO 
TOKENINFO_URL=http://localhost:9021//oauth2/tokeninfo
```
Notice the excess slash after the port number.

```
gyao@ip-172-xx-xxx-xxx:~$ curl -H "Authorization: Bearer xxx" "http://localhost:9021//oauth2/tokeninfo"
<a href="/oauth2/tokeninfo">Moved Permanently</a>.
```

```
gyao@ip-172-xx-xxx-xxx:~$ curl -H "Authorization: Bearer xxx" "http://localhost:9021/oauth2/tokeninfo"
{"error":"invalid_request","error_description":"Access Token not valid"}
```

**Comment by tuxlife** [comment]
@GJL which Taupage Version?

**Comment by tuxlife** [comment]
fixed with bc6024c68bd04923d3e0f3771b1d34540be51521
